hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -335,7 +335,9 @@ private OzoneManager(OzoneConfiguration conf) throws IOException,
         OZONE_OM_USER_MAX_VOLUME + " value should be greater than zero");
 
     if (omStorage.getState() != StorageState.INITIALIZED) {
-      throw new OMException("OM not initialized.",
+      throw new OMException("OM not initialized, current OM storage state: " +
+          omStorage.getState().name() + ". Please check if 'ozone om --init' "
```

Comment:
Can we rephrase "Please check if 'ozone om --init' command is executed before starting the OM service.",
Please ensure 'ozone om --init' command is executed once before starting the OM service.",

Reply:
Thanks @xiaoyuyao  for the review. Have addressed the comment now.